mods/d2k/rules/misc.yaml

```diff
@@ -152,3 +152,97 @@ wormspawner:
 	RenderEditorOnly:
 	BodyOrientation:
 	WormSpawner:
+
+upgrade.conyard:
+	Tooltip:
+		Name: Construction Yard Upgrade
+		Description: Upgrade for the Construction Yard
```

Comment:
Either go with something like `Description: Upgrade the Construction Yard\nto get access to advanced structures` or skip this.